src/Utilities/matrix_of_constraints.jl

```diff
@@ -219,6 +219,23 @@ This function should be implemented to be usable as storage of constants for
 """
 function set_from_constants end
 
+"""
+    modify_constants(constants, row::Integer, new_constant::T) where {T}
+    modify_constants(
+        constants,
+        rows::AbstractVector{<:Integer},
+        new_constants::AbstractVector{T},
+    ) where {T}
+
+Modify `constants` in-place to store `new_constant` in the `row` row, or
+This function returns an instance of the set `S` for which the constants where
```

Comment:
It seems to me that the function returns `nothing`

Reply:
Ha. It looks like I just didn't finish typing this

Reply:
Fixed